### DISCUSSION TITLE
How to use socket-io client in deno

### DISCUSSION THREAD

**Discussion by Himanshu922** [socketio/socket.io-deno#11] [Q&A]
Hi,
I want to test a socket.io server which user is running in node environment.
I provide user with some deno exe which will be running some test cases on that server but for that deno exe should act as a
socket.io client and make a connection with node socket.io server.
Please help me how can i achieve that?

**Comment by darrachequesne** [comment]
Hi! There is currently no client implementation in this package, but if there is enough interest we could implement/port it.

**Comment by khuongduybui** [comment]
I have seen people doing this
```ts
import { default as io } from "npm:socket.io-client@4.7.5";
```
